.github/workflows/publish-rust.yml

```diff
@@ -71,6 +62,14 @@ jobs:
       - name: Test
         run: pnpm zx ./scripts/rust/test.mjs "${{ inputs.package_path }}"
 
+      - name: Install cargo-semver-checks
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-semver-checks
+
+      - name: Check semver
+        run: pnpm rust:semver --manifest-path "${{ inputs.package_path }}/Cargo.toml" --release-type ${{ inputs.level }}
```

Comment:
This only works for crates `> 0.x`. Perhaps we should use the same as this [one](https://github.com/anza-xyz/pinocchio/blob/main/.github/workflows/publish.yml#L52-L81).

Reply:
Done!